src/styles/product-card.scss

```
@@ -8,12 +8,22 @@
   flex-direction: column;
   align-items: center;
   margin-bottom: $card-margin;
-  
+
   &__image {
     content: url(../images/imac.jpeg);
     width: 160px;
     height: 134px;
+    border: 0 solid transparent;
     margin: 30px 0;
+    transition: transform 1s;
+
+    &:hover {
+      border: 3px solid #f3f3f3;
```

Ти вже задав всі властивості border для елемента у 16 рядку. При наведенні треба змінити лише колір з прозорого на вказаний. Лаконічніше буде просто
```suggestion
      border-color: #f3f3f3;
```

---

Доречно, дякую!